src/storage/src/monitor/hummock_state_store_metrics.rs

```diff
@@ -105,6 +109,30 @@ impl HummockStateStoreMetrics {
         let iter_fetch_meta_duration =
             register_histogram_vec_with_registry!(opts, &["table_id"], registry).unwrap();
 
+        let read_req_bloom_filter_positive_counts = register_int_counter_vec_with_registry!(
+            "state_store_read_req_bloom_filter_positive_counts",
+            "Total number of read request that may false positive by bloom filters",
+            &["table_id", "type"],
+            registry
+        )
+        .unwrap();
+
+        let read_req_positive_but_non_exist_counts = register_int_counter_vec_with_registry!(
+            "state_store_read_req_positive_but_non_exist_counts",
+            "Total number of read request that bloom filter positive but results not exist",
+            &["table_id", "type"],
+            registry
+        )
+        .unwrap();
+
+        let read_req_check_bloom_filter_counts = register_int_counter_vec_with_registry!(
+            "state_store_read_req_check_bloom_filter_counts",
+            "Total number of read request that check bloom_filter with prefix hint",
```

Comment:
nits: "Total number of read request that checks bloom filter with a prefix hint"